ci/compare-versions/src/main.rs

```diff
@@ -129,7 +160,74 @@ fn print_selected_versions<'a>(
     output.to_string()
 }
 
-fn main() -> Result<()> {
+fn compare_rest() -> Result<()> {
+    let handle = ureq::builder().user_agent("fuelup").build();
+
+    let toml_resp = match handle.get(&CHANNEL_FUEL_LATEST_TOML_URL).call() {
+        Ok(r) => r
+            .into_string()
+            .expect("Could not convert channel to string"),
+        Err(e) => {
+            bail!(
+                "Unexpected error trying to fetch channel: {} - retrying at the next scheduled time",
+                e
+            );
+        }
+    };
+
+    let channel_doc = toml_resp
+        .parse::<Document>()
+        .expect("invalid channel.toml parsed");
+
+    let components = Components::collect_publishables()?;
+
+    for component in components
+        .iter()
+        .filter(|c| !["forc", "fuel-core"].contains(&c.name.as_str()))
```

Comment:
`"fuel-indexer"` should be added here as we maintain both `fuel-indexer` and  `forc-index`.

Reply:
Should `fuel-indexer` not be a downloadable component?

Reply:
Oh geez, I read it wrong. So sorry about that. 😓